specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,232 @@
+    pub updated_at: Option<u64>, // When token was last updated, Unix epoch in milliseconds
+    pub extra: Option<String>, // anything extra the SBT wants to store on-chain. Can be stringified JSON.
+    pub reference: Option<String>, // URL to an off-chain JSON file with more info.
+    pub reference_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of JSON from reference field. Required if `reference` is included.
+}
+
+
+trait SBT {
+    /**********
+    * QUERIES
+    **********/
+
+    /// get the information about specific token ID
+    fn sbt(&self, token_id: TokenId) -> Option<Token>;
+
+    /// returns total amount of tokens minted by this contract
+    fn sbt_total_supply(&self) -> U64;
+
+    /// returns total supply of SBTs for a given owner
+    fn sbt_supply_for_owner(&self, account: AccountId);
+
+    /// Query for sbt tokens
+    fn sbt_tokens(&self, from_index: Option<U64>, limit: Option<u32>) -> Vec<Token>;
+
+    /// Query sbt tokens by owner
+    fn sbt_tokens_for_owner(
+        &self,
+        account: AccountId,
+        from_index: Option<U64>,
+        limit: Option<u32>,
+    ) -> Vec<Token>;
+
+    /// Optional. If the SBT implementaiton assures that one account can have maximum one SBT
+    /// the the following function should be implemented.
+    /// Returns Some(Token) if an `account` owns an SBT, otherwise returns None.
+    fn sbt_token_for_owner(&self, account: AccountId) -> Option<Token> {}
+
+    /**********
+    * TRANSACTIONS
+    **********/
+
+    /// creates a new, unique token and assigns it to the `receiver`.
+    #[payable]
+    fn sbt_mint(&mut self, metadata: TokenMetadata, receiver: AccountId);
+
+    /// sbt_recover reassigns all tokens from the old owner to a new owner,
+    /// and registers `old_owner` to a burned addresses registry.
+    /// Must be called by operator.
+    /// Must provide 5 miliNEAR to cover registry storage cost. Operator should
+    ///   put that cost to the requester (old_owner), eg by asking operation fee.
+    #[payable]
+    fn sbt_recover(&mut self, from: AccountId, to: AccountId);
+
+    /// sbt_renew will update the expire time of provided tokens.
+    /// `expires_at` is a unix timestamp (in seconds).
+    #[payable]
+    pub fn sbt_renew(&mut self, tokens: Vec<TokenId>, expires_at: u64, memo: Option<String>);
+}
+```
+
+### Logs
+
+```typescript
+interface SbtEventLogData {
+  standard: "nepXXX";
+  version: "1.0.0";
+  event: "sbt_mint" | "sbt_recover" | "sbt_renew";
+  data: SbtMintLog[] | SbtRecoverLog[] | SbtRenew[];
+}
+
+// An event emitted when a new SBT is minted.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtMintLog {
+  owner: string;
+  tokens: string[];
+  memo?: string;
+}
+
+// An event emitted when existing SBTs are revoked and burned.
+// Arguments
+// * `owner`: "account.near"
+// * `tokens`: ["1", "abc"]
+// * `memo`: optional message
+interface SbtRevokeLog {
```

Comment:
SBTs are synonymous with non transferable nfts and are often used interchangeable. I think there should be some type of authority that can revoke a NFT, There currently isn't a standard for burning a NFT, maybe a burning of a NFT may include emitting the instance of deletion. Would love to see a separate implementation. Chloe the Dev touched on this https://gov.near.org/t/draft-non-fungible-token-burn-standard/28859. I think by default SBT owners should be given the option to revoke such a credential otherwise it will be used as a blacklist, despite who has the authority to revoke. Concerns around anti-spam protection.

Reply:
SBTs **are not** synonymous with non transferable nfts.
See discussion in other threads in this PR.

Reply:
> I think there should be some type of authority that can revoke

Yes, but that depends on the issuer. 
Also, revoking should not blacklist an account. Only soul transfer should do that.

Reply:
> SBTs are not synonymous with non transferable nfts.

@robert-zaremba Should you adjust the first sentence of the NEP then?

> Soulbound Tokens (SBT) are non transferable NFTs.

Reply:
updated